### ISSUE TITLE
Allow editor to select poster for related chapter

### ISSUE THREAD

**Issue by lottebelice** [linkedtv/editortool#39]
Right now, it's not possible to select a shot as a poster for related chapters.

**Comment by jblom** [comment]
the shots here are not meant for posters, but for setting the time period of the annotation (we don't use it in our current LinkedTV use cases). Also it does not make sense to select a shot from the current video (chapter) to depict a related chapter from another episode (or chapter from the same episode). Right?

**Comment by lottebelice** [comment]
Yes. I got that. My suggestion is to allow the editor to select a poster for the related chapter that is selected.

**Comment by jblom** [comment]
Well I never! Ok that does sound useful. FYI currently the poster you see is based on the starting frame of the related chapter.
